Gemfile.lock

```diff
@@ -236,7 +231,7 @@ GEM
     listen (3.0.8)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
-    loofah (2.3.0)
+    loofah (2.4.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.3.0 → 2.4.0

[_~~change-log~~, ~~source-code~~_]

:tada: Patched vulnerabilities:

- [CVE-2019-15587](https://nvd.nist.gov/vuln/detail/CVE-2019-15587)
  **Loofah XSS Vulnerability**

  
  URL: https://github.com/flavorjones/loofah/issues/171

  In the Loofah gem, through v2.3.0, unsanitized JavaScript may occur in sanitized output when a crafted SVG element is republished.






<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.3.0...v2.4.0).

These are the first 10 commits:
- (bcbd7b3) [update dev gemspec](https://github.com/flavorjones/loofah/commit/bcbd7b373176db3b4b2b249caaf196b625779c1b)
- (1908dc2) [update CHANGELOG with release date](https://github.com/flavorjones/loofah/commit/1908dc2defba6049bc17519c8b128d7030915204)
- (1bdf276) [formatting in README](https://github.com/flavorjones/loofah/commit/1bdf27600cf2433eb71fa542cce210663d8abef8)
- (a5bd819) [rufo formatting](https://github.com/flavorjones/loofah/commit/a5bd819f3ef13d5d4595106557c26169df2ef3a0)
- (0c6617a) [mitigate XSS vulnerability in SVG animate attributes](https://github.com/flavorjones/loofah/commit/0c6617af440879ce97440f6eb6c58636456dc8ec)
- (1d81f91) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/1d81f919bd29458a3b80966f9b6870b74b839dc9)
- (e323a77) [Merge pull request #172 from flavorjones/171-xss-vulnerabili…](https://github.com/flavorjones/loofah/commit/e323a776dd2755a837a67895eaa3cdae44495254)
- (83df303) [version bump to v2.3.1](https://github.com/flavorjones/loofah/commit/83df303aa14d58f76349b59e6917ae61ce011a83)
- (e5a9ae8) [ci: update concourse, add ruby 2.7 jobs](https://github.com/flavorjones/loofah/commit/e5a9ae8e8ee32ef0b026a90e6ce228cabb0f808e)
- (2699b61) [Allow CSS property: max-width](https://github.com/flavorjones/loofah/commit/2699b61a50e67adacabd1fc0990e8bfa69f63d1a)


</details>